templates/header.hbs

```diff
@@ -1,27 +1,32 @@
 <a class="skip-navigation" tabindex="1" href="#main-content">{{t 'skip_navigation' }}</a>
 
 <header class="header">
-  <div class="logo">
-    {{#link 'help_center'}}
-      <img src="{{settings.logo}}" alt="{{t 'home_page' name=help_center.name}}" />
-      {{#if settings.show_brand_name}}
-        <span aria-hidden="true">{{help_center.name}}</span>
-      {{/if}}
+  <div class="header-left">
+    <div class="logo">
+      <a href={{settings.xpro_link}} class="xpro-link"> 
+        <img src="{{settings.logo}}" class="site-logo" />
+      </a>
+    </div>
+    {{#link 'help_center' class="support-center"}}
+      SUPPORT CENTER
```

Comment:
In the Live theme https://xpro.zendesk.com/hc/en-us, this has a red branded color. Was there any reason to change it back to the default? 

<img width="382" alt="image" src="https://github.com/mitodl/mitxpro-zendesk-theme/assets/34372316/050afc9b-c499-4482-a89d-5d57e77662b8">

Reply:
The color will remain red if you haven't visited before or if you try to open the link in incognito mode.

Zendesk has introduced a configuration option (visited_link_color) to address this. The default value for this option was #9358B0, which did not align with our branding. Therefore, set the default value to match the branding of the mitxpro-zendesk-theme. However, the choice of default color is flexible and can be adjusted as needed. If there is disagreement regarding the default visited link color, we can make the default color for this new configuration to some other which can easily be changed from theme settings when required. Also providing screenshot of that feature of copenhagen_theme with master to verify:

![image](https://github.com/mitodl/mitxpro-zendesk-theme/assets/41048311/a61dc391-d96a-4351-943d-d6f23b5d4536)

Reply:
We should keep the production deployed theme and look as much possible. In this case, I think we can keep it to the red brand color of xPRO no matter visited or not because that's how the production theme is working